test: add test to verify stopping screenshare functionality

pinhaum opened the pull request:
### What does this PR do?
This pull request adds Playwright test coverage for the screenshare feature, specifically verifying that stopping screenshare restores the expected UI state.

### Motivation
Verifying that the UI updates as expected when stopping screenshare helps prevent regressions and improves the reliability of the screensharing experience.

pinhaum committed:
test: add stop screenshare test

pinhaum committed:
fix: rename stopScreenshareAndVerify to improve readability and clarity

pinhaum committed:
fix: update method name in spec

antonbsa reviewed (changes requested):
no need to add a separate test to check for stop sharing. add related actions + checks on the current "share screen" test:

- raname it to `start and stop screenshare`
- double-check function names to follow current pattern

pinhaum committed:
fix: join start and stop in the same share screen test

pinhaum commented:
made some refactoring to address the requested changes
- renamed the test to `Start and Stop Share screen` to no longer separate the actions
- removed some code from renamed `stopSharing()` method 

here`s a video from the playwright tests after the changes were made 
[video.webm](https://github.com/user-attachments/assets/d2bc54ab-31d5-46bb-b0b4-2600a8987799)

antonbsa reviewed (approved):
LGTM!
